sycl/test-e2e/ESIMD/vadd_usm_opqptr.cpp

```diff
@@ -5,7 +5,7 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
-// RUN: %{build} -Xclang -opaque-pointers -o %t.out
+// RUN: %{build} -o %t.out
```

Comment:
sorry another nit while we're here, we should actually just delete this. all this does is include another test `vadd_usm.cpp` and that will be running with opaque pointers now, so we're running the same test twice

Reply:
Sounds good.